engine/Assets/Scripts/UI/Dynamic/Panels/Configuring/Scoring/ZoneConfigPanel.cs

```diff
@@ -0,0 +1,328 @@
+using System;
+using System.Collections.Generic;
+using Synthesis.Gizmo;
+using Synthesis.UI;
+using Synthesis.UI.Dynamic;
+using UnityEditor.PackageManager;
```

Comment:
Either remove or add preprocessor definitions to remove this using statement when not inside the editor. Fails to compile otherwise.

Reply:
I don't think it's actually used so you're probably fine to just remove it